src/core_plugins/kibana/public/management/sections/objects/_objects.js

```diff
@@ -178,71 +193,85 @@ uiModules.get('apps/management')
           return;
         }
 
-        return new Promise((resolve) => {
-          confirmModal(
-            `If any of the objects already exist, do you want to automatically overwrite them?`, {
-              confirmButtonText: `Yes, overwrite all`,
-              cancelButtonText: `No, prompt me for each one`,
-              onConfirm: () => resolve(true),
-              onCancel: () => resolve(false),
-            }
-          );
-        })
-          .then((overwriteAll) => {
-            function importDocument(doc) {
-              const { service } = find($scope.services, { type: doc._type }) || {};
-
-              if (!service) {
-                const msg = `Skipped import of "${doc._source.title}" (${doc._id})`;
-                const reason = `Invalid type: "${doc._type}"`;
-
-                notify.warning(`${msg}, ${reason}`, {
-                  lifetime: 0,
+        const conflictedIndexPatterns = [];
+
+        function importDocument(doc) {
+          const { service } = find($scope.services, { type: doc._type }) || {};
+
+          if (!service) {
+            const msg = `Skipped import of "${doc._source.title}" (${doc._id})`;
+            const reason = `Invalid type: "${doc._type}"`;
+
+            notify.warning(`${msg}, ${reason}`, {
+              lifetime: 0,
+            });
+
+            return;
+          }
+
+          return service.get()
+            .then(function (obj) {
+              obj.id = doc._id;
+              return obj.applyESResp(doc)
+                .then(() => {
+                  return obj.save({ confirmOverwrite : false });
```

Comment:
Here we're preventing the ability to overwrite the objects, is that the desired behavior?

Reply:
Because I thought we wanted to remove that modal, I changed all the behavior to default to not overrwrite all, but it seems this is wrong regardless.